packages/typescript/internal/build_defs.bzl

```diff
@@ -14,7 +14,7 @@
 
 "TypeScript compilation"
 
-load("@build_bazel_rules_nodejs//internal/common:node_module_info.bzl", "NodeModuleInfo", "NodeModuleSources", "collect_node_modules_aspect")
+load("@build_bazel_rules_nodejs//internal/common:node_module_info.bzl", "NodeModuleInfo", "NodeModuleSourcesInfo", "collect_node_modules_aspect")
```

Comment:
@alexeagle Hmmmm. I wonder if we should be avoiding access internal paths in `@build_bazel_rules_nodejs` from npm published repositories such as `@npm_bazel_typescript` to avoid internal changes creating breakages like this one.

Reply:
Maybe add a pre-commit hook that checks for this condition as well when we fix it